modules/swagger-codegen/src/main/resources/JavaJaxRS/cxf-cdi/api.mustache

```diff
@@ -48,7 +52,7 @@ public class {{classname}}  {
     @ApiResponses(value = { {{#responses}}
         @ApiResponse(code = {{{code}}}, message = "{{{message}}}", response = {{{returnType}}}.class{{#returnContainer}}, responseContainer = "{{{returnContainer}}}"{{/returnContainer}}){{#hasMore}},{{/hasMore}}{{/responses}} })
     public Response {{nickname}}({{#allParams}}{{>queryParams}}{{>pathParams}}{{>headerParams}}{{>bodyParams}}{{>formParams}}{{#hasMore}}, {{/hasMore}}{{/allParams}}) {
-    	return delegate.{{nickname}}({{#allParams}}{{#isFile}}inputStream, fileDetail{{/isFile}}{{^isFile}}{{paramName}}{{/isFile}}, {{/allParams}}securityContext);
+    	return delegate.{{nickname}}({{#allParams}}{{#isFile}}fileInputStream, fileDetail{{/isFile}}{{^isFile}}{{paramName}}{{/isFile}}, {{/allParams}}securityContext);
```

Comment:
without this change there currently is a compile error in PetApi.java, as the parameter is declared with name fileInputStream instead of inputStream
``` java 
 @Multipart(value = "file", required = false) InputStream fileInputStream
```
So, the fix is to modify the returned parameter from inputStream to fileInputStream.
Of course also the parameter name in the method signature could be changed, pls let me know if you prefer it the other way.